codespan-reporting/src/term/renderer.rs

```diff
@@ -25,17 +25,18 @@ pub type SingleLabel<'diagnostic> = (LabelStyle, Range<usize>, &'diagnostic str)
 ///
 /// Locations are relative to the start of where the source code is rendered.
 pub enum MultiLabel<'diagnostic> {
-    /// Left top corner for multi-line labels.
-    ///
-    /// ```text
-    /// ╭
-    /// ```
-    TopLeft,
     /// Multi-line label top.
     ///
     /// ```text
     /// ╭────────────^
     /// ```
+    ///
+    /// Can also be rendered at the beginning of the line
+    /// if there is only whitespace before the label starts.
+    ///
+    /// /// ```text
+    /// ╭
+    /// ```
     Top(RangeTo<usize>),
```

Comment:
Is there any particular Reason why this is a `RangeTo` and not just a `usize`?

Reply:
No reason - I think I was trying to communicate that it was the range from the start of the line. We could switch it to a `usize` if that's easier though.

Reply:
I switched it and also the same thing in the `Bottom` variant. It's not necessarily easier now, but I added doc-comments so it should still be equally descriptive.